### PR TITLE
Add docs for v1.0

### PR DESCRIPTION
**Pull request by eketo-msft**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3090bb8](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/3090bb8f3bc7b9b7ba5882ae8d1d16a30b3e49b2)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 24 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3090bb8](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/3090bb8f3bc7b9b7ba5882ae8d1d16a30b3e49b2)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3090bb8](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/3090bb8f3bc7b9b7ba5882ae8d1d16a30b3e49b2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/directoryobjectpartnerreference.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/directoryobjectpartnerreference.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/organization.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/organization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C16%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902160033512313-3993%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=KxQ7M9bOpkjM3c4Izjh6G8GOwjYr99AIfYqKaYI%2BXeI%3D&st=2019-02-16T00%3A32%3A47Z&se=2019-03-19T00%3A37%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[107c8f6](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/107c8f6b23d812ad2280c644a9ff7f58b237f8ab)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 24 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[107c8f6](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/107c8f6b23d812ad2280c644a9ff7f58b237f8ab)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[107c8f6](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/107c8f6b23d812ad2280c644a9ff7f58b237f8ab)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/directoryobjectpartnerreference.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/directoryobjectpartnerreference.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/organization.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/organization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C16%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902160102341778-3993%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ZfAvVZNRHvqaVrCyGnPFXmf2rBfhQDhYuF%2FC9kGx014%3D&st=2019-02-16T01%3A01%3A50Z&se=2019-03-19T01%3A06%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d1cf949](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/d1cf9496cbb305c43a3f42736f98d3ad559cb8dd)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 24 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d1cf949](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/d1cf9496cbb305c43a3f42736f98d3ad559cb8dd)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

File: concepts/changelog.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 
 | **Change type** | **Version**   | **Description**                          |
 | :-------------- | :------------ | :--------------------------------------- |
+|Addition |beta and v1.0| Add the **createdDateTime** property on [organization](/graph/api/resources/organization?view=graph-rest-v1.0).|
```
Comment (Lauragra) [inline concepts/changelog.md]: I only see an update to the v1.0 topic in this PR. Do you need to update the beta topic?

File: concepts/changelog.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
...
+|Addition | beta | Added new resource type [directoryObjectPartnerReference](/graph/api/resources/directoryobjectpartnerreference?view=graph-rest-v1.0).|
```
Comment (Lauragra) [inline concepts/changelog.md]: This should be v1.0, not beta.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d1cf949](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/d1cf9496cbb305c43a3f42736f98d3ad559cb8dd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/directoryobject-getbyids.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/api/directoryobject-getbyids.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/directoryobjectpartnerreference.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/directoryobjectpartnerreference.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/organization.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/organization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C16%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902160111136835-3993%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ujNJzesrt0gNAgHeS7UuU%2BS3r82vxUJhu1hL0gsr1Ts%3D&st=2019-02-16T01%3A10%3A00Z&se=2019-03-19T01%3A15%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/changelog.md (hunk 2 above)
Comment (eketo-msft) [reply concepts/changelog.md]: already pushed a fix

File: concepts/changelog.md (hunk 1 above)
Comment (eketo-msft) [reply concepts/changelog.md]: beta is already updated:
https://docs.microsoft.com/en-us/graph/api/resources/organization?view=graph-rest-beta

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[aa89409](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/aa89409bf20d0d700e2dfbf53e2783b9166fe74d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[aa89409](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/aa89409bf20d0d700e2dfbf53e2783b9166fe74d)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[aa89409](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/aa89409bf20d0d700e2dfbf53e2783b9166fe74d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/directoryobject-getbyids.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/api/directoryobject-getbyids.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/directoryobjectpartnerreference.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/directoryobjectpartnerreference.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/organization.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/organization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902210346113659-3993%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OewbnWIz5GoyDuLlTRua84NCvY0gxSai%2FiU28cT8QDw%3D&st=2019-02-21T03%3A44%3A55Z&se=2019-03-24T03%3A49%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a0385f7](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/a0385f77a39ea2204b980ba7c406a3cf8898f680)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a0385f7](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/a0385f77a39ea2204b980ba7c406a3cf8898f680)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a0385f7](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/a0385f77a39ea2204b980ba7c406a3cf8898f680)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/directoryobject-getbyids.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/api/directoryobject-getbyids.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/directoryobjectpartnerreference.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/directoryobjectpartnerreference.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/organization.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/organization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C22%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902222239588811-3993%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gsSMAqDBwat5qmaAxY1rExL99wlHPVU0P6lbBrXK4W4%3D&st=2019-02-22T22%3A38%3A39Z&se=2019-03-25T22%3A43%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b43ebb0](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/b43ebb02d206ff1ec13a313461cdf218f4305fd6)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b43ebb0](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/b43ebb02d206ff1ec13a313461cdf218f4305fd6)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b43ebb0](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3993/commits/b43ebb02d206ff1ec13a313461cdf218f4305fd6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/directoryobject-getbyids.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/api/directoryobject-getbyids.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/directoryobjectpartnerreference.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/directoryobjectpartnerreference.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/organization.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/api-reference/v1.0/resources/organization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-20150215-updates/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C22%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902222240256731-3993%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wTbE2%2B88HXa2%2FR1eHwwE%2FqCShGG5flAIRFXIhA2skR0%3D&st=2019-02-22T22%3A42%3A53Z&se=2019-03-25T22%3A47%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**